ssz-rs/src/list.rs

```diff
@@ -240,6 +242,7 @@ where
             for (i, elem) in self.data.iter_mut().enumerate() {
                 let chunk = elem.hash_tree_root()?;
                 let range = i * BYTES_PER_CHUNK..(i + 1) * BYTES_PER_CHUNK;
+                // index is safe as long as Node::len() == BYTES_PER_CHUNK
```

Comment:
the `Node` type will always be `BYTES_PER_CHUNK` in size, so we can drop this comment, esp if you feel like that invariant is safely enforced elsewhere

Reply:
Definitely happy to drop it with `Node`'s length defined as `BYTES_PER_CHUNK`